client/me/application-passwords/index.jsx

```diff
@@ -96,7 +96,7 @@ class ApplicationPasswords extends Component {
 					onSubmit={ this.createApplicationPassword }
 				>
 					<FormFieldset>
-						<FormLabel htmlFor="application-name">{ translate( 'Application Name' ) }</FormLabel>
+						<FormLabel htmlFor="application-name">{ translate( 'Application name' ) }</FormLabel>
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.